Lib/test/test_asyncio/test_events.py

```diff
@@ -2912,11 +2881,6 @@ def test_get_event_loop_returns_running_loop2(self):
             loop = asyncio.new_event_loop()
             self.addCleanup(loop.close)
 
-            with self.assertWarns(DeprecationWarning) as cm:
```

Comment:
There were two tests here -- without and with `asyncio.set_event_loop(None)`. Now only one case is tested.

Reply:
fixed

Reply:
There is still only one test. Add `self.assertRaisesRegex...` before `asyncio.set_event_loop(None)`.

Reply:
sorry, that was an oversight fixed